js-jasmine/src/brie.js

```diff
@@ -0,0 +1,13 @@
+const Item = require('./item');
+
+class AgedBrie extends Item {
```

Comment:
I like that you've used the item class as the parent class and created the other item classes as children, really smart!